Added documentation for mode()

alfonsomhc opened the pull request:
This relates to issue  #9750

alfonsomhc committed:
Added documentation for mode()

This relates to issue  #9750

shoyer commented on pandas/core/frame.py:
```diff
@@ -4411,9 +4411,13 @@ def _get_agg_axis(self, axis_num):
+        Gets the mode(s) of each element along the axis selected. Empty if nothing
         has 2+ occurrences. Adds a row for each mode per label, fills in gaps
-        with nan.
+        with nan. Note that there could be multiple values returned for the selected
```
maybe add a paragraph break here?

shoyer commented on pandas/core/frame.py:
```diff
@@ -4411,9 +4411,13 @@ def _get_agg_axis(self, axis_num):
+        axis (when more than one item share the maximum frequency), which is the 
+        reason why a dataframe is returned. This means that if you want to impute 
+        missing values with the mode in a dataframe df, you can just do this: 
+        df.fillna(df.mode().ix[0])
```
two things:

1. If you "quote" code with two backticks `` on each time it will be formatted as code in the online documentation.
2. I would suggest using `.iloc` instead of `.ix`, because the later will fail if the index consists of integers.

alfonsomhc committed:
Updated documentation for mode()

Improved after review. Backtick for code mode, new paragraph for the note, iloc instead of ix for integer indices.

alfonsomhc commented:
Thanks for the feedback, shoyer. I have added a new commit. Hopefully it is fine now :)

shoyer commented on pandas/core/frame.py:
```diff
@@ -4411,9 +4411,14 @@ def _get_agg_axis(self, axis_num):
+        Gets the mode(s) of each element along the axis selected. Empty if nothing
         has 2+ occurrences. Adds a row for each mode per label, fills in gaps
-        with nan.
+        with nan. 
```
You actually need a full empty line to make a new paragraph :)

alfonsomhc committed:
Updated documentation for mode()

added new line to make the note in a new paragraph

alfonsomhc commented:
Thanks, shoyer. Hopefully it is ok now?

shoyer commented:
Merged, thanks!
